cryptography/src/androidMain/kotlin/com/wire/kalium/cryptography/ProteusClientCryptoBoxImpl.kt

```diff
@@ -25,63 +31,67 @@ class ProteusClientCryptoBoxImpl constructor(rootDir: String) : ProteusClient {
         return false
     }
 
-    override suspend fun openOrCreate() {
+    /**
+     * Create the crypto files if missing and call box.open
+     * this must be called only one time
+     */
+    override suspend fun openOrCreate() = withContext(coroutineContext) {
```

Comment:
This IO operation should be on IO dispatcher

Reply:
@ohassine the IO police